Model/ResourceModel/Invoice.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * @category    ClassyLlama
+ * @copyright   Copyright (c) 2017 Classy Llama Studios, LLC
+ */
+
+namespace ClassyLlama\AvaTax\Model\ResourceModel;
+
+class Invoice extends \Magento\Framework\Model\ResourceModel\Db\AbstractDb
+{
+    protected function _construct()
```

Comment:
Database field names could be defined as constants and referenced instead of statically typed strings throughout the code.

An example of this to this on: \ClassyLlama\AvaTax\Model\ResourceModel\Queue